pageserver/src/metrics.rs

```diff
@@ -2337,13 +2337,15 @@ macro_rules! redo_bytes_histogram_count_buckets {
 pub(crate) struct WalIngestMetrics {
     pub(crate) bytes_received: IntCounter,
     pub(crate) records_received: IntCounter,
+    pub(crate) records_observed: IntCounter,
     pub(crate) records_committed: IntCounter,
     pub(crate) records_filtered: IntCounter,
```

Comment:
did you mean to remove the old ones here and below in WAL_INGEST?

Reply:
Not yet. The Vanilla protocol still uses the received, committed, filtered metrics. Once we finish roll-out of the interpreted protocol, we can remove them.